projects/storefrontlib/src/lib/cms/guards/cms-page.guard.ts

```diff
@@ -34,6 +36,7 @@ export class CmsPageGuard implements CanActivate {
           this.routingService.go(['notFound']);
         }
       }),
+      tap(([_, pageContext]) => this.cmsI18N.loadNamespaces(pageContext)),
```

Comment:
I'm thinking, if we need that `getPageComponentTypes(pageContext)` always, for routes, translations (and in future, for guard-in-component implementation) maybe it would be good to add it to `combineLastest` above. We won't need that additional call with subscription hidden in `loadNamespaces`